### PR TITLE
Complete serialize support

### PR DESCRIPTION
**Pull request by jayvdb**
Version bumps & publishing will be done after this is approved & merged.

### CODE REVIEW

File: butane/Cargo.toml [hunk 1]
```diff
@@ -25,9 +26,10 @@ tls = ["butane_core/tls"]
 uuid = ["butane_core/uuid", "butane_codegen/uuid"]
 
 [dependencies]
-butane_codegen = { path = "../butane_codegen", version = "0.5.1-alpha.6", registry = "franklin-ai-rosalind" }
-butane_core = { path = "../butane_core", version = "0.5.2-alpha.5", registry = "franklin-ai-rosalind" }
-
+#butane_codegen = { path = "../butane_codegen", version = "0.5.1-alpha.6", registry = "franklin-ai-rosalind" }
+#butane_core = { path = "../butane_core", version = "0.5.2-alpha.5", registry = "franklin-ai-rosalind" }
+butane_codegen = { path = "../butane_codegen" }
+butane_core = { path = "../butane_core" }
```
Comment (joaommartins) [inline butane/Cargo.toml:32]: Need to be updated for release.

File: butane_core/src/fkey.rs [hunk 2]
```diff
@@ -23,8 +23,7 @@ use fake::{Dummy, Fake, Faker};
 ///   blog: ForeignKey<Blog>,
 ///   ...
 /// }
-///
-#[derive(Default)]
+#[derive(Clone, Default, Deserialize, Serialize)]
```
Comment (joaommartins) [inline butane_core/src/fkey.rs:26]: Does this correctly dereferences the ForeignKey struct into the inner type when serialising?

Comment (jayvdb) [reply butane_core/src/fkey.rs:26]: yup

Comment (joaommartins) [reply butane_core/src/fkey.rs:26]: Awesome, thought you'd have to do a manual impl.

**Review by joaommartins — APPROVED** [review]
lGtm

**Pull request merged**